### DISCUSSION TITLE
Authentication on sub-folders

### DISCUSSION THREAD

**Discussion by Neryss** [sixem/ivfi-php#25] [Questions] [answered]
Hello!
First of all, thanks for the awesome tool you've made, I love it!

I wanted to ask if there was a way to enable authentication on specific sub-folders only? Or does it need to be tinkered through the script?

Thanks in advance, have a good day!

**Comment by sixem** [comment, marked as answer]
Hey!

As of right now there isn't a way to configuring that through the options, no.

One option is to configure the web server to serve an alternative index file with an alternative config file that uses authentication for certain sub-folders, otherwise the script will have to be tinkered with.

However, you are not the first person to ask about this, so I'll see if I can add an option for this in the future when I have some time!

**Reply by Neryss** [reply]
Thank you for the answer, no problem with that, I'll serve an alternative config for now then. I'll keep an eye on future updates!

Thanks again for this tool, have a good day!

**Comment by Pantyhose-X** [comment]
pin password lock
